index.bs

```diff
@@ -614,6 +614,30 @@ information about the platform version on which a given [=user agent=] is execut
 [=Structured Header=] whose value MUST be a [=structured header/string=]
 [[!RFC8941]].
 
+[=User agents=] SHOULD return a version with a platform-specific format that allows differentiating significant platform versions by running these steps:
+    1. Let |platformVersionComponentList| be a [=list=].
+    1. Use per-platform logic:
+        * If the platform is Android:
+            1. Let |platformReturnedVersionString| be the result of querying the OS's `android.os.Build.VERSION.RELEASE` string.
+            1. Parse |platformReturnedVersionString|, storing the first three decimal integers delimited by U+002E (FULL STOP). If |platformReturnedVersionString| does not contain three integer values, replace the missing components with "0".
+            1. Append all three version components in the order they were read to |platformVersionComponentList|.
+        * If the platform is macOS:
+            1. Let |platformReturnedVersionString| be the result of querying the `UIDevice` return by `currentDevice` and reading its `systemVersion`.
+            1. Parse |platformReturnedVersionString|, storing the first three decimal integers delimited by U+002E (FULL STOP). If |platformReturnedVersionString| does not contain three integer values, replace the missing components with "0".
+            1. Append all three version components in the order they were read to |platformVersionComponentList|.
+        * If the platform is Linux:
+            1. Let |platformReturnedVersionString| be the result of querying the `release` string in the `utsname` struct returned by the `uname` API.
+            1. Parse |platformReturnedVersionString|, storing the first three decimal integers delimited by U+002E (FULL STOP). If |platformReturnedVersionString| does not contain three integer values, replace the missing components with "0".
+            1. Append all three version components in the order they were read to |platformVersionComponentList|.
+        * If the platform is macOS:
+            1. Let |platformReturnedVersionString| be the result of querying the `NSOperatingSystemVersion`.
```

Comment:
nit: missing `API` at end of sentence?

Reply:
I made the pointer more clear to the API that Chromium is using and read more like the iOS description.